mailgun/mailgun.go

```diff
@@ -67,8 +67,12 @@ func httpError(ctx context.Context, w http.ResponseWriter, err error) bool {
 // https://documentation.mailgun.com/en/latest/user_manual.html#securing-webhooks
 func validSignature(ctx context.Context, req *http.Request, apikey string) bool {
 	h := hmac.New(sha256.New, []byte(apikey))
-	io.WriteString(h, req.FormValue("timestamp"))
-	io.WriteString(h, req.FormValue("token"))
+	if _, err := io.WriteString(h, req.FormValue("timestamp")); err != nil {
+		log.Log(ctx, errors.Wrap(err, "Issue with validSignature when using io.WriteString"))
+	}
+	if _, err := io.WriteString(h, req.FormValue("token")); err != nil {
+		log.Log(ctx, errors.Wrap(err, "Issue with validSignature when using io.WriteString"))
+	}
```

Comment:
These should be ignored to eliminate noise; the package documentation guarantees it will never return an error:
https://pkg.go.dev/hash#Hash

Reply:
changed to ignore the error since I personally don't think we need the error message, and if you're ok with ignoring it in these cases where we don't really care if it errors anyways.